lib/set.rb

```diff
@@ -0,0 +1,120 @@
+class Set
```

Comment:
Given your definition "A set represents one round of the game", I think `Round` or `GameRound` is a better name for this class

Reply:
Agree! i called it Round